### DISCUSSION TITLE
How to specify service names for virtual database

### DISCUSSION THREAD

**Discussion by 2310652493** [apache/skywalking#11735] [Q&A] [answered]
How to specify service names for virtual database
![image](https://github.com/apache/skywalking/assets/88769942/cfbf4612-622e-46d6-8d4d-8da42e2a3c74)

**Comment by wu-sheng** [comment, marked as answer]
This has been documented, https://skywalking.apache.org/docs/main/next/en/setup/service-agent/virtual-database/

**Reply by 2310652493** [reply]
This means that I cannot specify the server name in the Skywalking configuration as it is generated through program analysis

**Reply by wu-sheng** [reply]
Yes, APM represents your system, not going to change randomly. Make the visualization as close to reality as possible.

**Reply by 2310652493** [reply]
Thank
